### PR TITLE
Updated Litmus Checker to run as Non-Root User

### PR DESCRIPTION
**Pull request by gdsoumya**
Signed-off-by: Soumya Ghosh Dastidar <gdsoumya@gmail.com>

Updated Litmus Checker to run as Non-Root User

### CODE REVIEW

File: custom/litmus-checker/Dockerfile [hunk 1]
```diff
@@ -14,5 +14,7 @@ LABEL maintainer="LitmusChaos"
 
 #Install kubectl
 #Copy binaries from build stage
-COPY --from=builder /output/checker /
-ENTRYPOINT ["./checker"]
\ No newline at end of file
+RUN addgroup -S litmus && adduser -S litmus -G litmus && mkdir -p /var/run && chown -R litmus:litmus /var/run /tmp
```
Comment (rajdas98) [inline custom/litmus-checker/Dockerfile:17]: using `addgroup` and `chown` also requires  permission

Check chaos-operator dockerfile for ref- https://github.com/litmuschaos/chaos-operator/blob/master/build/Dockerfile

**Review by rajdas98 — CHANGES_REQUESTED** [review]

File: custom/litmus-checker/Dockerfile [hunk 2]
```diff
@@ -14,5 +14,7 @@ LABEL maintainer="LitmusChaos"
 
 #Install kubectl
 #Copy binaries from build stage
-COPY --from=builder /output/checker /
```
Comment (rajdas98) [inline custom/litmus-checker/Dockerfile:17]: You copying to root director. Just have to change the path

File: custom/litmus-checker/Dockerfile (hunk 1 above)
Comment (gdsoumya) [reply custom/litmus-checker/Dockerfile]: It is already built into the image, root access is not required to run the image once it is built.

File: custom/litmus-checker/Dockerfile (hunk 2 above)
Comment (gdsoumya) [reply custom/litmus-checker/Dockerfile:17]: The image enters as a root user, we have to add a new user and group first and then we can just change the ownership

**Review by ksatchit — APPROVED** [review]

**Review by uditgaurav — APPROVED** [review]
LGTM

**Review by ispeakc0de — APPROVED** [review]

File: custom/litmus-checker/Dockerfile (hunk 2 above)
Comment (gdsoumya) [reply custom/litmus-checker/Dockerfile:17]: The path is also change to /var/run

**Review by rajdas98 — APPROVED** [review]

**Pull request merged**